### PR TITLE
Feature/update UI missing data unit test

### PR DESCRIPTION
**Pull request by selectionator**
It works. It just works. But it can't merge 😢

### CODE REVIEW

File: HotelPriceScout/Data/Interface/Dashboard.cs [hunk 1]
```diff
@@ -102,11 +102,12 @@ public void UpdateUiMissingDataWarning(BookingSite bookingSite)
                     {
                         if(price.Price == 0)
                         {
-                            warnings += $"On date: {price.Date} hotel: {hotelName}, with roomtype: {roomtype}|";
+                            warnings = $"On date: {price.Date} hotel: {hotelName}, with roomtype: {roomtype.Capacity}|";
```
Comment (WassawRoki) [inline HotelPriceScout/Data/Interface/Dashboard.cs]: chagne to +=

File: Tests/DashboardTest.cs [hunk 2]
```diff
@@ -53,17 +56,73 @@ public void Test_If_ArrowDecider_Returns_Correct_Value(string expected, int mark
...
+            Dashboard dashboard = new Dashboard();
+            Dictionary<string, string> hotelStrings = new Dictionary<string, string>()
+            {
+                {"hotel", "tag" }
+            };
+            BookingSite bookingSite = new BookingSite("hotel", "single", "https://www.url.com", hotelStrings);
+
+            bookingSite.DataScraper.StartScraping(10);
+
+            bookingSite.HotelsList.First().RoomTypes.First().Prices.First().Price = 0;
```
Comment (WassawRoki) [inline Tests/DashboardTest.cs:87]: det sku en god lang dreng den her haha :D men fin

**Review by WassawRoki — APPROVED** [review]

File: Tests/DashboardTest.cs (hunk 2 above)
Comment (selectionator) [reply Tests/DashboardTest.cs:87]: deez

File: Tests/DashboardTest.cs [hunk 3]
```diff
@@ -53,17 +49,73 @@ public void Test_If_ArrowDecider_Returns_Correct_Value(string expected, int mark
...
+
+            dashboard.UpdateUiMissingDataWarning(bookingSite);
+
+            string dash = dashboard.WarningMessage.First().ListofWarnings;
```
Comment (Sarmisuper) [inline Tests/DashboardTest.cs:68]: kald den noget andet

File: Tests/DashboardTest.cs [hunk 4]
```diff
@@ -53,17 +49,73 @@ public void Test_If_ArrowDecider_Returns_Correct_Value(string expected, int mark
...
+            };
+            BookingSite bookingSite = new BookingSite("bookingSite", "single", "https://www.url.com", hotelStrings);
+
+            bookingSite.DataScraper.StartScraping(10);
+
+            bookingSite.HotelsList.First().RoomTypes.First().Prices.First().Price = 0;
+
+            dashboard.UpdateUiMissingDataWarning(bookingSite);
+
+            string dash = dashboard.WarningMessage.First().ListofWarnings;
```
Comment (Sarmisuper) [inline Tests/DashboardTest.cs:91]: kald den noget andet

**Review by Sarmisuper — CHANGES_REQUESTED** [review]
Do a theory

**Review by Sarmisuper — APPROVED** [review]
lgtm

**Pull request merged**